lib/feature/pokemon_details_page.dart/widgets/evolution_tab/pokemon_evolution_tab.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex_async_redux/api/models/evolution_chain_model.dart';
+import 'package:pokedex_async_redux/feature/pokemon_details_page.dart/widgets/evolution_tab/widgets/pokemon_evolution_tab_widget.dart';
+
+class EvolutionTab extends StatelessWidget {
+  const EvolutionTab({required this.thisPokeEvo});
+  final EvolutionChain thisPokeEvo;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: EdgeInsets.all(5),
+      child: SingleChildScrollView(
+        child: Wrap(
+          children: [
+            Container(
+              height: 80,
+              width: 80,
+              child: PokemonEvolutionWidget(thisSpecies: thisPokeEvo.evolutionOne),
+            ),
+            thisPokeEvo.evolutionTwo.isNotEmpty
```

Comment:
`if(condition)`
`   Widget()`

doesn't make sense to return a sizedBox. Either add the widget if it meets the condition or don't add it.